src/odemis/driver/powerctrl.py

```diff
@@ -44,46 +46,62 @@ class PowerControlUnit(model.PowerSupplier):
     communication with the PCU firmware.
     '''
 
-    def __init__(self, name, role, port, pin_map=None, delay=None, init=None, ids=None,
+    def __init__(self, name, role, port, pin_map=None, delay=None, init=None,
+                 ids: Optional[List[str]] = None,
+                 excluded_ids: Optional[List[str]] = None,
                  termination=None, check_power=True, **kwargs):
-        '''
+        """
         port (str): port name
         pin_map (dict of str -> int): names of the components
           and the pin where the component is connected.
         delay (dict str -> float): time to wait for each component after it is
             turned on.
         init (dict str -> boolean): turn on/off the corresponding component upon
             initialization.
-        ids (list str): EEPROM ids expected to be detected during initialization.
+        ids (None or list[str]): EEPROM ids expected to be detected during initialization.
+        All the IDs must be found. If not, another device is search for.
```

Comment:
what do you mean with "another device is search for"?

Reply:
Search for another PCU. I've updated the docstring to clarify this text.